crates/typst-library/src/model/link.rs

```diff
@@ -30,35 +30,35 @@ use crate::text::TextElem;
 /// ]
 /// ```
 ///
-/// # Hyphenation
-/// If you enable hyphenation or justification, by default, it will not apply to
-/// links to prevent unwanted hyphenation in URLs. You can opt out of this
-/// default via `{show link: set text(hyphenate: true)}`.
+/// # ハイフネーション
+/// ハイフネーションや両端揃えを有効にしていても、意図しないURL中のハイフネーションを防ぐため、
+/// デフォルトではリンクには適用されません。
+/// これを無効化するには、`{show link: set text(hyphenate: true)}`を使用します。
 ///
-/// # Syntax
-/// This function also has dedicated syntax: Text that starts with `http://` or
-/// `https://` is automatically turned into a link.
+/// # 構文
+/// この関数には専用の構文もあります。
+/// `http://`や`https://`で始まるテキストは、自動的にリンクに変換されます。
 #[elem(Show)]
 pub struct LinkElem {
-    /// The destination the link points to.
+    /// リンクの遷移先。
     ///
-    /// - To link to web pages, `dest` should be a valid URL string. If the URL
-    ///   is in the `mailto:` or `tel:` scheme and the `body` parameter is
-    ///   omitted, the email address or phone number will be the link's body,
-    ///   without the scheme.
+    /// - Webページにリンクする場合、`dest`は有効なURL文字列である必要があります。
+    ///   `mailto:`または `tel:`スキームを含むURLが指定され、
```

Comment:
【コメント】
「スキーム」と単にいうよりは「URLスキーム」と言ったほうが親切かもしれません。

```suggestion
    ///   `mailto:`や`tel:`のURLスキームを含むURLが指定され、
```